web/libs/editor/src/core/Registry.ts

```diff
@@ -111,6 +132,13 @@ class _Registry {
   getPerRegionView(tag: string | number, mode: string | number) {
     return this.perRegionViews[tag]?.[mode];
   }
+
+  addCustomTag(tag: string, definition: CustomTag) {
+    this.addTag(tag.toLowerCase(), definition.model, definition.view);
+    this.addObjectType(definition.model);
```

Comment:
It looks like we won't be able to add something different from the object tag to the Custom tags.
Is it expectable behaviour?